roles/install/tasks/main.yml

```diff
@@ -0,0 +1,114 @@
+---
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2022, Felix Fontein
+
+- name: Gather required information on localhost
+  when: sops_install_on_localhost
+  ansible.builtin.setup:
+    gather_subset:
+      - '!all'
+      - '!min'
+      - architecture
+      - distribution
+      - distribution_major_version
+      - distribution_version
+      - os_family
+  delegate_to: localhost
+  delegate_facts: true
+  run_once: true
+
+- name: Show system information
+  ansible.builtin.debug:
+    msg: |-
+      Architecture: {{
+        hostvars['localhost' if sops_install_on_localhost else inventory_hostname].ansible_facts.architecture
```

Comment:
Also, there's going to be a new [avoid-dot-notation: Prefer bracket notation vs. dot notation for value retrieval](https://github.com/ansible/ansible-lint/issues/2174) ansible-lint rule, but meh.

Reply:
> Would it be better to set `{{ hostvars['localhost' if sops_install_on_localhost else inventory_hostname] }}` as a separate variable and reference that each time (e.g. `{{ vars_list.ansible_facts.architecture }}` if that var is named vars_list) to make this more readable and DRY?

I thought about that, but didn't want to use `set_fact` and then forgot about it... I will change that.

> Also, there's going to be a new [avoid-dot-notation: Prefer bracket notation vs. dot notation for value retrieval](https://github.com/ansible/ansible-lint/issues/2174) ansible-lint rule, but meh.

I hope that will never happen, and if it will, it's a rule I will ignore :)